src/lua/httpc.lua

```diff
@@ -439,7 +515,14 @@ curl_mt = {
 -- Export
 --
 local http_default = http_new()
-local this_module = { new = http_new, }
+local this_module = {
```

Comment:
AFAIU, the `encoders` field on an http client instance is part of the public API (the documentation request misses it, though, see the comment above).

But how to legally access encoders of the default http client instance? I guess `require('http.client').encoders` should return it.